src/Helpers/LiFiData.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+
+pragma solidity ^0.8.17;
+
+/// @title LiFiData
```

Comment:
`LiFiData `==> `LiFi Data`

Reply:
Well, to be honest I never understand why we split the title up. For me the title is the title (aka the same what I would use after "contract .... {}"

What is your opinion?